crates/autopilot/src/domain/cow_amm.rs

```diff
@@ -0,0 +1,33 @@
+use {
+    cow_amm::Amm,
+    primitive_types::H160,
+    std::{collections::HashSet, sync::Arc},
+};
+
+/// Cow AMM
+/// The list of owners is initialized with the values specified in the autopilot
+/// configuration file, and it is extended with all the CoW AMM owners which are
+/// deployed by the CoW AMM helper contract
+#[derive(Clone)]
+pub struct CowAmm {
```

Comment:
We actually don't necessarily need this, but I thought it would be nice to have a `domain` structure containing all these methods.